tests/Fixture/Events/FoundryEventListener.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the zenstruck/foundry package.
+ *
+ * (c) Kevin Bond <kevinbond@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zenstruck\Foundry\Tests\Fixture\Events;
+
+use Symfony\Component\EventDispatcher\Attribute\AsEventListener;
+use Zenstruck\Foundry\Event\AfterInstantiate;
+use Zenstruck\Foundry\Event\AfterPersist;
+use Zenstruck\Foundry\Event\BeforeInstantiate;
+use Zenstruck\Foundry\Tests\Fixture\Entity\EntityForEventListeners;
+
+final class FoundryEventListener
+{
+    #[AsEventListener]
+    public function beforeInstantiate(BeforeInstantiate $event): void
```

Comment:
I'm wondering if we cannot still introduce an attribute `#[BeforeInstantiate(Address::class)]`?

the DX will be a little bit better